diff -urN gtksourceview-1.0.1.orig/configure.in gtksourceview-1.0.1/configure.in
--- gtksourceview-1.0.1.orig/configure.in	2004-04-19 09:32:22.952137040 +0200
+++ gtksourceview-1.0.1/configure.in	2004-04-19 09:32:34.249419592 +0200
@@ -94,7 +94,7 @@
 fi
 
 # I18N stuff
-ALL_LINGUAS="az ar be bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gu he hi hr hu id it ja ko lt mk mn ml ms nl no pa pl pt pt_BR ru sk sq sr sr@Latn sv ta tr uk vi zh_CN zh_TW"
+ALL_LINGUAS="az ar be bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gu he hi hr hu id it ja ko lt mk mn ml ms nb nl pa pl pt pt_BR ru sk sq sr sr@Latn sv ta tr uk vi zh_CN zh_TW"
 GETTEXT_PACKAGE=gtksourceview-1.0
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [Package name for gettext])
